apps/nextjs-website/src/helpers/breadcrumbs.helpers.ts

```diff
@@ -60,3 +60,21 @@ export function pageToBreadcrumbs(
     ...(paths || []),
   ];
 }
+
+export function solutionPageToBreadcrumbs(
```

Comment:
i think is not necessary to add a new function but can be used instead `pageToBreadcrumbs` above in this file

Reply:
I think it's a ghost function